docs/migration/migration1110to1120.adoc

```diff
@@ -22,3 +22,7 @@ A változtatások nem eredményeznek átállási munkálatokat, visszafelé komp
 ==== Átállás
 * Amennyiben a régi működést szeretnénk alkalmazni, azaz ha exception-t dob a redis stream consumer,
 akkor ne történjen `XACK`, akkor a stream konfigjában fel kell venni a `coffee.redisstream.{0}.consumer.manualAck` kulcsot `true` értékkel.
+
+=== coffee-module-mp-restclient
+* `DefaultBaseExceptionResponseExceptionMapper` fel lett készítve "application/octet-stream" válaszra.
+Ilyenkor megpróbálja a streamben válaszolt `BaseExceptionResultType`-ot json vagy xml szerint felparszolni. 
```

Comment:
Átállásról leírás?

Reply:
nem kell semmi, nem is erinti az elozo dolgokat mert eddig null-al tert vissza

Reply:
Ennyit ilyenkor is szoktunk írni: `A változtatások nem eredményeznek átállási munkálatokat, visszafelé kompatibilis.`

Reply:
bele raktam